tests/lib/rules/comment-directive.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * @fileoverview Tests for comment-directive rule.
+ * @author Toru Nagashima
+ */
+
+'use strict'
+
+// -----------------------------------------------------------------------------
+// Requirements
+// -----------------------------------------------------------------------------
+
+const assert = require('assert')
+const path = require('path')
+const Module = require('module')
+const eslint = require('eslint')
+
+// -----------------------------------------------------------------------------
+// Tests
+// -----------------------------------------------------------------------------
+
+// Initialize linter.
+const linter = new eslint.CLIEngine({
+  parser: 'vue-eslint-parser',
+  parserOptions: {
+    ecmaVersion: 2015
+  },
+  plugins: ['vue'],
+  rules: {
+    'vue/comment-directive': 'error',
+    'vue/no-parsing-error': 'error',
+    'vue/no-duplicate-attributes': 'error'
+  },
+  useEslintrc: false
+})
+
+describe('comment-directive', () => {
+  // Preparation.
+  // Make `require("eslint-plugin-vue")` loading this plugin while this test.
+  const resolveFilename = Module._resolveFilename
+  before(() => {
+    Module._resolveFilename = function (id) {
+      if (id === 'eslint-plugin-vue') {
+        return path.resolve(__dirname, '../../../lib/index.js')
+      }
+      return resolveFilename.apply(this, arguments)
+    }
+  })
+  after(() => {
+    Module._resolveFilename = resolveFilename
+  })
+
+  describe('eslint-disable/eslint-enable', () => {
+    it('disable all rules if <!-- eslint-disable -->', () => {
+      const code = `
+        <template>
+          <!-- eslint-disable -->
```

Comment:
Could you perhaps add few more examples with nested/multiline elements like e.g.:
```vue
<template>
  <ul>
    <li v-for="item in items" <!-- eslint-disable-line -->
      :class="item.class"
      :title="item.title"
      <!-- eslint-disable-next-line -->
      title="test"
    />
  </ul>
</template>
```

Reply:
It's just a syntax error in HTML.